### PR TITLE
Feature/player score

### PR DESCRIPTION
**Pull request by shitongs**
Added player score js api documentation

### CODE REVIEW

File: source/includes/score.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+##Creating a Score
+
+```javascript 
+// Initialize score with specified element. Spawned marginal scores will appear around that element
+var coinsScore = score.create(this, 'coins');
+
+// Initialize score without element. Spawned marginal scores will appear in front of the camera or at the specified location
+var points = score.create('points');
+```
+
```
[the remark this answers is not in the document]
Comment (gdamron) [reply source/includes/score.md]: After looking at the whole thing I see you do that later. I think some of these earlier comments should still be pulled out of the code block. I'd say the (hand-wavy) rule of thumb is that if it's explaining what the API does, put it after a `> ` outside of a code block. If it's describing how part of an example works, a comment is fine.

File: source/includes/score.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+log.info(coinsScore.getScoreFormat());   	//prints '+ {score} pts'
+log.info(coinsScore.isMuted()); 	 	//prints true
+log.info(coinsScore.getDisplayVine); 	 	//prints the vine element
+
+```
+The score manager has reference to all created scores. All score information can be accessed and modified through the manager.
+
+##Spawned Score Location
+
+>Scores declared with attached Element
```
[the remark this answers is not in the document]
Comment (shitongs) [reply source/includes/score.md]: Thank you for all of the catches! Should all be fixed now :)

File: source/includes/score.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+//sets the color of margin score display to be red
+coinsScore.color(new col4(1,0,0,1))
+
+//sets margin score display font size to be 90
+coinsScore.fontSize(90)
+
+//sets score display offset
+//When there is an element attached, this is the local offset
+//When there is none attached, this is the global offset
+coinsScore.offset(new vec3(3,3,0))
```
Comment (jsundra) [inline source/includes/score.md]: Similarly, `coinsScore.offset(vec3(3, 3, 0))`

File: source/includes/score.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+
+//defines the string fromatting of the spawned visualization. {score} will be replaced with the marginal score
+//if marginal score is 2, the visualization spawned is 'You received 2 points!'
+coinsScore.scoreFormat('You received {score} points!');
+
+//mutes sound fx
+coinsScore.mute(true);
+
+//sets the vine that displays the updated total score
+coinsScore.setDisplayVine(this.parent.findOne(..'coins-score'));
```
Comment (jsundra) [inline source/includes/score.md]: Small string typo `..'coins-score'` -> `'..coins-score'`

File: source/includes/score.md [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+> Access to specific score and make modifications
+
+```javascript
+const score = require('score-preview');
+
+var coinsScore = score.getType('coins');
+coinsScore.score(10)
+	.fontSize(100)
+	.color(new col4(0,0,0,1))
+	.offset(new vec3(1,2,3))
```
Comment (jsundra) [inline source/includes/score.md]: Same as above re: C#

File: source/includes/score.md [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+	.duration(3)
+	.randomPosRange(1.5)
+	.scoreFormat('+ {score} pts') 
+	.mute(true)
+	.setDisplayVine(this.findOne(..'coins-score'));
```
Comment (jsundra) [inline source/includes/score.md]: Same as above re: string formatting

**Pull request merged**